authcp/views.py

```diff
@@ -133,3 +133,16 @@ def activation(request, key):
         # Display : error message
         already_active = True
     return render(request, 'authcp/activation.html', locals())
+
+
+# save user who is registered with google or facebook
+def save_profile(backend, user, response, *args, **kwargs):
+
+    if backend.name == "github":
+        print "Github user {} is logged in".format(user)
```

Comment:
I don't know anything about our log mechanism. We should not print to console like that. I will remove it, but i would like to make a conversation about log at first.